src/dos/program_mount.cpp

```diff
@@ -462,7 +462,9 @@ void MOUNT::AddMessages() {
 	        "\n"
 	        "Notes:\n"
 	        "  - '-t overlay' redirects writes for mounted drive to another directory.\n"
-	        "  - '-usecd #' gives direct access to a CD-ROM drive (needed for CD audio).\n"
+	        "  - '-usecd ID' gives direct access to a CD-ROM drive.\n"
+	        "This is needed for CD audio (only supported on Windows and Linux).\n"
+	        "Run 'mount -cd' to find out the list of valid IDs.\n"
```

Comment:
Nice, thank you 👍🏻

Reply:
One thing, can you please indent these two new lines with the first line of the bullet point? So just add 4 spaces in front, if I'm counting them right 😄